packages/zowe-explorer/src/extension.ts

```diff
@@ -873,7 +886,7 @@ function initSubscribers(context: vscode.ExtensionContext, theProvider: IZoweTre
     }
 }
 
-function getSelectedNodeList(node: IZoweTreeNode, nodeList: IZoweTreeNode[]): IZoweTreeNode[] {
+export function getSelectedNodeList(node: IZoweTreeNode, nodeList: IZoweTreeNode[]): IZoweTreeNode[] {
```

Comment:
@KutluOzel-b this function seems to be a duplicate of getSelectedNodeList() in shared/utils?  We should only have one of these functions especially if they are exactly the same and I would suggest having it stay in the shared/utils file vs adding more to the extension.ts file.

Reply:
@JillieBeanSim Thanks for the warning. This is caused by pulling from main branch. Keeping it in shared/utils